src/test/java/org/junitpioneer/jupiter/RetryingTestExtensionTests.java

```diff
@@ -216,6 +255,33 @@ void failsOnlyOnFirstInvocation() {
 			}
 		}
 
+		@RetryingTest(value = 3, onExceptions = IllegalArgumentException.class)
+		void failsOnlyOnFirstInvocationWithExpectedException() {
+			EXECUTION_COUNT++;
```

Comment:
I know it was there before, but why is execution count upper camel case? It's not a constant?